### DISCUSSION TITLE
Server enviroments with Container

### DISCUSSION THREAD

**Discussion by Klaus51978** [itzg/docker-minecraft-server#2713] [Q&A] [answered]
Hello. I want to start the server in "creative" mode for my daughter. Unfortunately, this doesn't work if I enter "MODE" as the name and "creative" as the value for the enviroments.
Then when I connect to my client, it's not creative mode.
![grafik](https://github.com/itzg/docker-minecraft-server/assets/55620118/8d6529f1-5614-486d-91c4-e791291b2037)

**Comment by itzg** [comment]
Hmm, that seems right as far as the container can control. You can set `DUMP_SERVER_PROPERTIES` to "true" to confirm what properties file gets created and check that this line is present during the startup logs

```
gamemode=creative
```

If it's a world that has already been generated then you might need to change the gamemode with the command: https://minecraft.wiki/w/Game_mode#Changing_game_mode

**Reply by Klaus51978** [reply, marked as answer]
I have stop the server. delete the folder "world". env = Mode - Creative. Start Server... everything is fine ;-)
